pkgs/applications/misc/collision/gi-crystal/gi-crystal.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile	2023-07-08 09:55:34.286726723 +0300
++++ b/Makefile	2023-07-08 09:56:06.302214795 +0300
+@@ -1,7 +1,7 @@
+ .PHONY: test just-test oldlibs-test generator test-binding libtest compare recompare doc
+ 
+ test: test-binding
+-	./bin/spec $(RSPEC_OPTS)
++	sh bin/spec $(RSPEC_OPTS)
```

Comment:
Why not patch the shebang instead?

Reply:
This patch was removed in https://github.com/NixOS/nixpkgs/commit/9c1cd513a09b5380ac72043e690390c54590a6bb